### ISSUE TITLE
[Bug]: Camera keeps recentering every few seconds

### ISSUE THREAD

**Issue by Gelthir** [Turing-Complete-Game/Suggestions-and-Issues#201]
### Discord Thread

https://discord.com/channels/828292123936948244/1278075628066705461

### What happened?

0.1129 Beta, Linux X11 (Debian 12)

This happens in levels, in the component editor, and in the level map. There doesn't seem to be a fixed period to it; sometimes it happens within 1s of the last recenter, sometimes 10s.

Every time the camera recenters, this gets printed to stderr:
```
ERROR: (Node not found: "/root/level/map/components/Null" (absolute path attempted from "/root/f").)
   at: get_node (scene/main/node.cpp:1322)
```
Repros with a clean profile, even in the very first Crude Awakening level before the level map has been unlocked.

It seems to happen less frequently if I don't move the mouse (but still happens sometimes). Possibly there's some invisible thing that mousing over causes it.

### Version

0.1129 Beta

### What OSes are you seeing the problem on?

Linux

**Comment by Stuffe** [comment]
Does this still happen? Any way to reproduce?

**Comment by Arnavion** [comment]
It still happens in 0.1137 Beta. Repro is as I said in the OP - load any level, move the camera somewhere other than the default, then move the mouse around for 10 seconds or so.

**Comment by Stuffe** [comment]
fixed next patch

**Comment by Arnavion** [comment]
Confirmed fixed in 0.1147 Beta
